src/jit/flowgraph.cpp

```diff
@@ -18969,6 +18969,14 @@ void Compiler::fgSetBlockOrder()
             // JIT64 does.
             genInterruptible = true;
         }
+
+        GenTreePtr tailCall                 = nullptr;
+        bool fastTailCallsOnly              = true;
+        bool tailCallsConvertibleToLoopOnly = false;        
+        if (!hasTailCalls && block->endsWithTailCall(this, fastTailCallsOnly, tailCallsConvertibleToLoopOnly, &tailCall))
```

Comment:
Should we make this arm/arm64 specific so that it does not bloat the size of GCInfo on x64?